crates/eips/src/eip1898.rs

```diff
@@ -379,14 +379,14 @@ impl From<BlockNumberOrTag> for BlockId {
     }
 }
 
-impl From<B256> for BlockId {
-    fn from(block_hash: B256) -> Self {
+impl From<BlockHash> for BlockId {
+    fn from(block_hash: BlockHash) -> Self {
         Self::Hash(RpcBlockHash { block_hash, require_canonical: None })
     }
 }
 
-impl From<(B256, Option<bool>)> for BlockId {
-    fn from(hash_can: (B256, Option<bool>)) -> Self {
+impl From<(BlockHash, Option<bool>)> for BlockId {
+    fn from(hash_can: (BlockHash, Option<bool>)) -> Self {
         Self::Hash(RpcBlockHash { block_hash: hash_can.0, require_canonical: hash_can.1 })
     }
```

Comment:
Note, left this as changed to open discussion - I've generally opted for not using the alias for `From` impl's, curious what is preferred in this case:

- Change all where useful
- Do not change any